scriptured-prayer-components/src/components/PrayerDeck.tsx

```diff
@@ -11,12 +11,15 @@ import "swiper/css/pagination";
 import "~/swiper.css";
 import { useApi } from "~/hooks";
 import { CardResponse } from "~/api/models/responses";
+import { CategoryResponse } from "~/api/models/responses";
 import { Card } from "./Card";
+import { FocusCard } from "./FocusCard";
 
 function PrayerDeck() {
   const api = useApi();
   const { name } = useParams();
   const [cards, setCards] = useState<CardResponse[]>([]);
+  const [category, setCategory] = useState<CategoryResponse>();
 
   useEffect(() => {
```

Comment:
Merge both `useEffect` instances into one. Based on how you are handling errors inside `FocusCard.tsx`, you need both `category` and `cards` to be ready before you can display anything useful to the user.
```suggestion
  useEffect(() => {
    (async () => {
      Promise.all([
        api.cards.all({ category__name: name }),
        api.categories.all(),
      ])
        .then(([_cards, categories]) => {
          setCards(_cards);
          // todo: find by id instead of name
          setCategory(categories.find((c) => c.name === name));
        })
        .catch((error) => console.error(error));
    })();
  }, []);
```